frontend/pages/admin/OrgSettingsPage/OrgSettingsPage.tsx

```diff
@@ -62,22 +62,24 @@ const OrgSettingsPage = ({ params, router }: IOrgSettingsPageProps) => {
       // send all formUpdates.agent_options because diff overrides all agent options
       diff.agent_options = formUpdates.agent_options;
 
-      configAPI
-        .update(diff)
-        .then(() => {
+      try {
+        try {
```

Comment:
🤨 not saying I'm smarter than someone with jets for brains but, I think this can be unrolled to just one try/catch.